src/hub/dataload/sources/chebi/chebi_upload.py

```diff
@@ -23,15 +21,13 @@
 class ChebiUploader(BaseDrugUploader):
 
     name = "chebi"
-    #storage_class = storage.IgnoreDuplicatedStorage
+    # storage_class = storage.IgnoreDuplicatedStorage
     storage_class = storage.RootKeyMergerStorage
     __metadata__ = {"src_meta": SRC_META}
-    keylookup = MyChemKeyLookup(
-        [('inchikey', 'chebi.inchikey'),
-         ('drugbank', 'chebi.xrefs.drugbank'),
-         ('chebi', 'chebi.id'),
-         ],
-        copy_from_doc=True)
+    keylookup = MyChemKeyLookup([('inchikey', 'chebi.inchikey'),
+                                 ('drugbank', 'chebi.xrefs.drugbank'),
+                                 ('chebi', 'chebi.id')], copy_from_doc=True)
+
     # See the comment on the ExcludeFieldsById for use of this class.
```

Comment:
I'm sure I'm missing it, but what comment is being referenced here? I'm assuming we want to have some inline comment on how/why we populate `exclusion_ids`...

Reply:
A document with an ID from `exclusion_ids` would have a long list for one or more of the following fields:

- `chebi.xrefs.intenz`
- `chebi.xrefs.rhea`
- `chebi.xrefs.uniprot`
- `chebi.xrefs.sabio_rk`
- `chebi.xrefs.patent`

`ExcludeFieldsById` acts like a filter to truncate the length of such long lists to 1,000.